admin/console.rst

```diff
@@ -51,6 +51,29 @@ This reduces loading times greatly.
 
 .. include:: /admin/console-rails-shell.include.rst
 
+.. warning:: **Safe Mode**
```

Comment:
I think ``warning`` is a bit agressive for a hint / note for people not solving their dependencies.
Maybe a ``hint`` would be an fitting alternative